qml/SettingsPage.qml

```diff
@@ -75,6 +75,14 @@ Page {
                     checked: settings ? settings.showWeekNumber : false
                 }
             }
+
+            AbstractButton {
+                id: weekNumberButton
+                anchors.fill: parent
+                onClicked: {
+                    weekCheckBox.checked = !weekCheckBox.checked;
+                }
+            }
```

Comment:
Could you not just use the onClicked of the ListItem for this? or is there a specific reason you didn't?

Reply:
I didn't even think to. I could test it and see, I guess.

Reply:
It works. Updated the PR for this.